admin-dev/themes/new-theme/webpack.config.js

```diff
@@ -4,42 +4,112 @@ var ExtractTextPlugin = require("extract-text-webpack-plugin");
 
 module.exports = {
   entry: [
-    "./js/theme.js"
+    require.resolve('tether'),
```

Comment:
Why don't you import all of these from `theme.js` instead of having so many entry points?